### PR TITLE
Fixed routing from project to user

### PR DESCRIPTION
**Pull request by slemmon**
This PR addresses:
Issue when navigating to a Modite from a Project details view
Details view card styling
Project details view list items styling
Local time not updating on the Modite details view
Adjusted map center for details view so the individual map point is positioned more naturally in the map view

And maybe a few other things, but I've since forgotten what all I tackled
Oh, yeah, and dropped element focus styling since it looked weird more often than not

### CODE REVIEW

File: src/components/App/theme.css [hunk 1]
```diff
@@ -165,3 +165,7 @@ a {
   text-decoration: none;
   color: var(--ion-color-modus-charcoal);
 }
+
+*:focus {
+  outline: transparent;
```
Comment (grgur) [inline src/components/App/theme.css]: a11y killer

File: src/components/ModiteList/styles.module.css [hunk 2]
```diff
@@ -171,3 +171,32 @@
   top: 20px;
   left: 0px;
 }
+
+.detailsView > .moditeListWrap > div > div[class="List"] {
+  opacity: 0;
+}
+
+.detailsView ~ div[class*="MapCt"] div[class*="MapEl"] {
```
Comment (grgur) [inline src/components/ModiteList/styles.module.css]: what does this `div[class*="MapCt"]` dirty magic do?

File: src/components/App/theme.css (hunk 1 above)
Comment (slemmon) [reply src/components/App/theme.css]: We want to modify this for this app?

File: src/components/ModiteList/styles.module.css (hunk 2 above)
Comment (slemmon) [reply src/components/ModiteList/styles.module.css]: "Fun" way to modify the map position of a peer / child element when the ModiteList switches to a detail view for Modite or Project

File: src/components/App/theme.css (hunk 1 above)
Comment (grgur) [reply src/components/App/theme.css]: I kinda believe in a11y. I mean, you could be having the Sun at your screen, right? That's also accessibility

Comment (slemmon) [reply src/components/App/theme.css]: ok.  fixed

**Pull request merged**